### ISSUE TITLE
Consider deleting the demo folder when publishing

### ISSUE THREAD

**Issue by leinelissen** [brigade/react-waypoint#279]
Hi there!

I was just doing an audit of our Node application, and I noticed this package on my storage graph. The demo folder is being packed into the NPM package, causing 1.8MB of redundant code to be downloaded whenever this package is installed. Please consider deleting it when publishing, or only publishing a subset of files.

Thanks!

**Comment by trotzig** [comment]
Good catch! A line in an `.npmignore` file should be enough here, right? Any help is appreciated.

**Comment by leinelissen** [comment]
@MatthewHerbst damn that's quick. In way before I got a change haha.
